docs: update README based on rename (#1289)
openedx/frontend-app-authoring: README.rst

frontend-app-authoring
######################

|license-badge| |status-badge| |codecov-badge|

Purpose
*******

> This implements most of the frontend for **Open edX Studio**, allowing authors to create and manage edit content such as courses and content libraries.

Author: ```suggestion
This implements most of the frontend for **Open edX Studio**, allowing authors to create and edit courses, libraries, and their learning components.
```
